src/platforms/react-native/sourcemaps.mdx

```diff
@@ -16,6 +16,19 @@ However, if you use custom values for your release other than the version includ
 
 For events sent from Sentry to correctly be attributed to a release and subsequently its source maps, **both the `release` and `dist` values will need to be set.** You can set these values in the call to `init`, as discussed in our [Releases & Health content](/platforms/react-native/configuration/releases/).
 
+<Note>
+
+Before going ahead and disabling the automatic source maps upload, you may set the `release` and `dist` as environment variables, `SENTRY_RELEASE` and `SENTRY_DIST` respectively. The script that does the automatic source maps upload will use those values instead the default ones and they will match the given `release` and `dist`.
```

Comment:
Should we add here the problem users are trying to avoid by doing this. i've suggested a final sentence, but please update for accuracy. 

```suggestion
Before disabling the automatic source maps upload, you can set the `release` and `dist` as environment variables, `SENTRY_RELEASE` and `SENTRY_DIST` respectively. The script that performs the automatic source maps upload will use those values instead the default ones, and they will match the given `release` and `dist`. This ensures that auto-generated release names are accepted.
```

Reply:
@imatwawana since it's not an auto-generated release anymore, I'd rather change to:

This ensures that the custom release and dist names are used.

wdyt?

Reply:
I like it!

Reply:
Oh maybe just add the word "values" or "variables" — whichever makes sense. So "This ensures that the given custom `release` and `dist` values are used.

Reply:
values is a good option, thanks